chpl/chpl-service/src/main/java/gov/healthit/chpl/sharedstore/SharedStoreDAO.java

```diff
@@ -56,6 +56,15 @@ public void remove(String type, List<String> keys) {
         entityManager.createQuery(delete).executeUpdate();
     }
 
+    @Transactional()
+    public void removeByDomain(String domain) {
```

Comment:
Is this called from anywhere? In the diff here, I see it as a new method but I cannot find it referenced again in any of the other diffs. (Certainly could have missed it)

Reply:
This is called in `SharedStoreProvider.removeAll()`
![image](https://user-images.githubusercontent.com/24817202/183940713-974b2304-bcda-4812-9b36-325069d29637.png)